### ISSUE TITLE
sense key in WNGT is not correct for adj

### ISSUE THREAD

**Issue by lwmlyy** [getalp/UFSAC#6]
For example, the correct sense key for 'emergent.a.02' is 'emergent%5:00:00:nascent:00', but it is labeled as 'emergent%3:00:00:nascent:00'. There are hundreds of them which are mislabeled in terms of '3' or '5' for adj.

wn30_key="emergent%3:00:00:nascent:00;emerging%3:00:00:nascent:00"

**Comment by loic-vial** [comment]
Hi,
This was already discussed in another issue, I copy/paste my answer:

About the adjective satellite marker in WordNet, we detected an inconsistency in the original data of version 3.0 between the WordNet raw "dict" directory, where adjective satellite have the "%5" marker, and the "glosstag" directory, where they all have the "%3" marker.

I remember seeing somewhere in the WordNet 3.0 doc, or maybe in a changelog from WN 2.1 to WN 3.0, a paragraph saying that adjective satellite will now have the same "%3" marker as all the other adjectives, starting with version 3.0.

In online version 3.1, there is no more %5 marker, so we decided to replace all %5 marker in all  corpora to %3, so there is no more confusion: an adjective always has a %3 marker in UFSAC.
There is no clash with existing "%3" adjectives so it's just simpler and less confusing like that.
